pages/_meta.json

```diff
@@ -8,6 +8,7 @@
   "sounds": "Sound",
   "codecs": "Codecs",
   "text-and-translations": "Text and Translations",
+  "misc": "Miscellaneous",
```

Comment:
The miscelaneous section was removed, just use the top level (`pages/` instead of `pages/misc` for this page, put it below text and translations.